frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -11,48 +10,132 @@ interface TopNavBarProps {
+        bottom: '-6px',
+        left: 0,
+        width: '100%',
+        height: '3px',
+        backgroundColor: '#0EE094',
+    },
+    '&:hover::after': {
+        content: '""',
+        position: 'absolute',
+        bottom: '-6x',
+        left: 0,
+        width: '100%',
+        height: '3px',
+        backgroundColor: '#0ee094',
+    },
+});
 
-export const TopNavBar: FCWOC<TopNavBarProps> = ({ children, className }) => {
-    const user = useCurrentUser()
-    const hideBanner = user.isResearcher
+const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'));
+const AccountLinks = loadAsync('Account Links', () => import('./account-links'));
+export const TopNavBar: React.FC<TopNavBarProps> = ({ className }) => {
+    const user = useCurrentUser();
+    const hideBanner = user.isResearcher;
+    const isMobile = useIsMobileDevice();
 
     return (
-        <nav className={cx('navbar', 'navbar-light', className)}>
-            <div className="navbar-dark bg-dark py-1">
-                <Container>
-                    <Box justify="between" align="center" gap padding={{ vertical: 'default' }}>
-                        <NavbarLogoLink />
-                        {children}
-                        <Box gap="xlarge" align="center">
-                            <Link to="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>
-                                Dashboard
-                            </Link>
-                            <DesktopResearcherLinks />
+        <nav
+            className={cx('navbar', 'navbar-light', className)}
+            style={{
+                backgroundColor: colors.navy,
+                color: 'white',
+                height: '80px',
+            }}
+        >
+            <div style={{ height: '80px', backgroundColor: colors.navy }}>
+                <Container style={{ height: '80px' }}>
+                    <Group
+                        justify="space-between"
+                        align="center"
+                        style={{
+                            width: '100%',
+                            height: '80px',
+                        }}
+                    >
+                        <Group>
+                            <NavbarLogoLink />
+                        </Group>
+                        <Group
+                            justify="center"
+                            align="center"
+                            gap="40px"
+                            style={{
+                                height: '80px',
+                                padding: '26px 0px 26px 0px',
+                            }}
+                        >
+                            <NavbarStyledLink to="/studies">
+                                <Flex align="center" gap="5px">
+                                    All Studies
+                                </Flex>
+                            </NavbarStyledLink>
+                            {!isMobile &&
```

Comment:
I think this should be handled differently. All the links should not appear when it is in mobile view. All studies still appear. It can be made into a component and mapped, to simplify it.